primitives/src/lib.rs

```diff
@@ -178,3 +151,14 @@ pub type FeeSplit<R, B1, B2> = SplitFeesByRatio<R, FeeSplitRatio, B1, B2>;
 /// https://w3f-research.readthedocs.io/en/latest/polkadot/Token%20Economics.html#-2.-slow-adjusting-mechanism
 pub type SlowAdjustingFeeUpdate<R> =
 	TargetedFeeAdjustment<R, TargetBlockFullness, AdjustmentVariable, MinimumMultiplier>;
+
+/// Unique identifiers for pallets.
```

Comment:
W: I would move the pallet_id module into its own file. I know it does not have a lot of information, but it is easier to find. This lib.rs file feels like a big pot of stuff.

Reply:
7857d6d